### PR TITLE
Fix issue related to missing module imports in persist generated clients

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose

Fixes:[3593](https://github.com/ballerina-platform/ballerina-standard-library/issues/3593)

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_21/target/persist_db_scripts.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+);
+
+CREATE TABLE AidPackageOrderItem (
+	id INT NOT NULL AUTO_INCREMENT,
+	needId INT UNIQUE,
+	CONSTRAINT FK_AIDPACKAGEORDERITEM_MEDICALNEED_0 FOREIGN KEY(needId) REFERENCES MedicalNeed(needId),
+	quantity INT NOT NULL,
+	totalAmount INT NOT NULL,
+	PRIMARY KEY(id)
+);
```
Comment (daneshk) [inline persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_21/target/persist_db_scripts.sql:20]: add extra line at the end

**Comment by daneshk** [comment]
@kaneeldias please review

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java [hunk 2]
```diff
@@ -370,22 +370,19 @@ public static SyntaxTree generateClientSyntaxTree(Entity entity, ArrayList<Impor
...
         String className = entity.getEntityName();
         boolean inclusions = false;
         boolean manyRelation = false;
         if (entity.getRelations().size() != 0) {
             relationsEnum = new Enum(String.format(ENUM_NAME, className));
             for (Relation relation : entity.getRelations()) {
+                if (!hasTime) {
+                    for (FieldMetaData fieldMetaData :relation.getRelatedFields()) {
+                        if (fieldMetaData.getFieldType().contains(BalFileConstants.KEYWORD_TIME)) {
+                            hasTime = true;
```
Comment (kaneeldias) [inline persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java:382]:
```suggestion
                            hasTime = true;
                            break;
```

**Comment by kaneeldias** [comment]
LGTM

File: persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_21/target/persist_db_scripts.sql (hunk 1 above)
Comment (sahanHe) [reply persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_21/target/persist_db_scripts.sql]: I added a new line but this file is a Autogenerated one by the compiler plugin

**Review by daneshk — APPROVED** [review]

**Pull request merged**